rest/resource-server/src/docs/asciidoc/components.adoc

```diff
@@ -293,3 +293,20 @@ include::{snippets}/should_document_get_usedbyresource_for_components/curl-reque
 
 ===== Example response
 include::{snippets}/should_document_get_usedbyresource_for_components/http-response.adoc[]
+
+[[resources-mysubscriptions-components]]
+==== Listing mysubscriptions components with fields
+
+A `GET` request will list all of the service's recent components.
```

Comment:
Add proper description related to new endpoint.

Reply:
done.